src/main/resources/db/migration/V5__add_tag.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tags (
+  uuid        UUID PRIMARY KEY,
+  created_at  TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
We'll want the application layer to set the timestamp (for example, [`Mapper.toDatasetRow()`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/service/mappers/Mapper.java#L188)). Meaning, we can remove defaulting the timestamp to `CURRENT_TIMESTAMP`.

Reply:
fixed.